.github/workflows/build-plugin-archive.yml

```
@@ -202,16 +215,27 @@ jobs:
           composer global require inpsyde/wp-translation-downloader
           composer wp-translation-downloader:download
 
+      - name: Run PHP Scoper
```

The official name is "PHP-Scoper"; see https://github.com/humbug/php-scoper.
```suggestion
      - name: Run PHP-Scoper
```